azure-votes-redis/azure-vote-managed-redis.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: azure-vote
+---
+apiVersion: resources.azure.com/v1api20200601
+kind: ResourceGroup
+metadata:
+  name: aso-redis-demo
+  namespace: azure-vote
+spec:
+  location: westcentralus
+---
+apiVersion: cache.azure.com/v1api20201201
+kind: Redis
+metadata:
+  name: azure-vote-redis
+  namespace: azure-vote
+spec:
+  location: westcentralus
+  owner:
+    name: aso-redis-demo
+  sku:
+    family: C
+    name: Basic
+    capacity: 0
+  enableNonSslPort: true
+  minimumTlsVersion: "1.2"
+  redisConfiguration:
+    maxmemory-delta: "10"
+    maxmemory-policy: allkeys-lru
+  redisVersion: "6"
+  operatorSpec: 
+    secrets:
+      hostName:
+        name: redis-hostname
```

Comment:
Are redis hostnames dynamically generated so that 2 different users of this sample running it in two different subscriptions won't get any issue when both trying to create a Redis with the same name (`azure-vote-redis`)?

Reply:
Added instructions to set app name as env variable and then use the envsubst command.